src/index.js

```diff
@@ -142,18 +159,21 @@ export class WT {
     return {
       dimensions: {
         width: this.context.innerWidth,
-        height: this.context.innerHeight,
+        height: this.context.innerHeight
       },
       agent: this.context.navigator.userAgent,
-      rts: (new Date()).valueOf(),
+      rts: new Date().valueOf(),
+      wvt: retrieveFromQueryString(
+        this.context.location && this.context.location.search
```

Comment:
instead of passing in this value, can we just reference `this` inside the `retrieveFromQueryString` method?

Reply:
believe `this` will be undefined in `retrieveFromQueryString`.